stable/prometheus-operator/templates/_helpers.tpl

```diff
@@ -1,137 +1,83 @@
 {{/* vim: set filetype=mustache: */}}
-{{/*
-Expand the name of the chart.
-*/}}
+{{/* Expand the name of the chart. This is suffixed with -alertmanager, which means subtract 13 from longest 63 available */}}
 {{- define "prometheus-operator.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 50 | trimSuffix "-" -}}
```

Comment:
this will cause a statefulset creation failure, we should leave 50 until this PR get merged 
https://github.com/coreos/prometheus-operator/pull/1362